js/src/sheet.ts

```diff
@@ -136,7 +137,13 @@ let SheetModel = widgets.DOMWidgetModel.extend({
                 cell_data.options['renderer'] = cell.get('renderer') || cell_data.options['renderer'];
                 cell_data.options['readOnly'] = cell.get('read_only') || cell_data.options['readOnly'];
                 cell_data.options['source'] = cell.get('choice') || cell_data.options['source'];
-                cell_data.options['format'] = cell.get('format') || cell_data.options['format'];
+                if (cell.get('numeric_format') && cell.get('type') == 'numeric') {
+                    cell_data.options['numericFormat'] = {'pattern': cell.get('numeric_format')};
+                }
+                if (cell.get('date_format') && cell.get('type') == 'date') {
+                    cell_data.options['correctFormat'] = true;
+                    cell_data.options['dateFormat'] = cell.get('date_format') || cell_data.options['dateFormat'];
+                }
```

Comment:
This is a change in handsontable?

Reply:
Yes, the format attribute disappeared and was replaced by `numericFormat` and `dateFormat`

Reply:
Ok, this is a breaking change, so we should up the minor number.